site/components/src/main/java/uk/nhs/digital/freemarker/utils/package-info.java

```diff
@@ -0,0 +1,8 @@
+/**
+ * A collection of generic helper Freemarker functions.
+ *
+ * Freemarker template usage example:
+ *  <#assign inArray="uk.nhs.digital.freemarker.utils.InArray"?new() />
+ *  <#if inArray(options.status?lower_case, ['complete'])>
+ */
+package uk.nhs.digital.freemarker.utils;
```

Comment:
:heart_eyes:

:+1: for package documentation - I don't see it (or apply :disappointed: myself) often enough.